Jenkinsfile

```diff
@@ -914,6 +914,10 @@ def pullContainerImages(String mavenArgs) {
 		// Cannot use a foreach loop because then Jenkins wants to serialize the iterator,
 		// and obviously the iterator is not serializable.
 		for (int i = 0; i < containerImageRefs.length; i++) {
+			if (containerImageRef.startsWith('docker.elastic.co')) {
```

Comment:
+1 would work right now. If you want to be generic, you could do something like:

```
if ( containerImageRef has two slashes && does not start with `([^/]*\.)?docker\.io` )
```

Also just so you know (since I know you don't like duplicating code between Jenkinsfiles), it would be perfectly acceptable to do the filtering [in the groovy code](https://github.com/hibernate/hibernate-search/blob/e78126f71f9cdd4973789be29602a7e7e4ae4285/pom.xml#L953-L993).